docs/reference/connectors-kibana/elastic-managed-llm.md

```diff
@@ -0,0 +1,30 @@
+---
+navigation_title: "Elastic Managed LLM"
+---
+
+# Elastic Managed LLM
+
+Elastic provides a default LLM optimized for product features.
+Details of the currently used model are available in the [model card](https://raw.githubusercontent.com/elastic/kibana/refs/heads/main/docs/reference/resources/Elastic_Managed_LLM_model_card.pdf).
+
+::::{note}
+The default LLM may change in the future based on evaluations of performance, security, and accuracy.
+::::
+
+## Region and hosting
+
+The Elastic Managed LLM is currently hosted in AWS US regions, beginning with `us-east-1`.
+Expansion to additional cloud service providers (CSPs) and regions may occur in the future.
```

Comment:
Let's remove "Expansion to additional cloud service providers (CSPs) and regions may occur in the future."